### PR TITLE
A4A: Add intro cards to the overview page

### PR DESCRIPTION
**Pull request by nateweller**
Resolves https://github.com/Automattic/jetpack-genesis/issues/291

## Proposed Changes

* Changes `tips` naming to `intro-cards` to match Jetpack Manage verbiage.
* Add the intro cards.

## Testing Instructions

* Review the intro cards via the `/overview` page on desktop and mobile.
* Validate the card content against the P2: p2-pfunGA-AR

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

## Screenshots

Screenshot 2024-03-19 at 3 40 36 PM

### CODE REVIEW

File: client/a8c-for-agencies/sections/overview/body/intro-cards/style.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.intro-cards {
+	.dot-pager__page {
+		p,
+		li {
+			font-size: rem(16px);
+			font-weight: 400;
+			line-height: 24px;
+			color: var(--studio-gray-80);
```
Comment (andrii-lysenko) [inline client/a8c-for-agencies/sections/overview/body/intro-cards/style.scss]: Similar to this: https://github.com/Automattic/wp-calypso/pull/88733#discussion_r1533386721 suggestion we could replace `--studio` vars

**Review by andrii-lysenko — APPROVED** [review]
This looks good to me codewise and works as described. I'm wondering though, what should happen when hitting `Done` button, but we can address it in the future.

**Comment by nateweller** [comment]
> I'm wondering though, what should happen when hitting Done button, but we can address it in the future.

I've removed the `includeDoneButton` prop from the `DotPager` component, easy fix! Thanks!

File: client/a8c-for-agencies/sections/overview/body/intro-cards/style.scss (hunk 1 above)
Comment (nateweller) [reply client/a8c-for-agencies/sections/overview/body/intro-cards/style.scss]: This is good to know – I've removed these `color` styles entirely as the default body `--color-text` variable is already equal to the `--studio-gray-80` value :+1:

**Pull request merged**